spec/setup_test_chitter.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+p "Setting up test chitter..."
+
+def setup_test_chitter
+connection = PG.connect(dbname: 'chitter_test')
+
+# Clear the bookmarks table
```

Comment:
the rubric talks about a Rake task to start on an empty database - could be worth exploring..?